### PR TITLE
Removed unnecessary titles on word clouds.

### PR DESCRIPTION
**Pull request by indiah444**
- Removed duplicated titles in `5_Word_Clouds.py`

### CODE REVIEW

**Comment by ayahusseini** [comment]
Good idea and makes the dashboard a lot more readable/less cluttered.

File: dashboard/pages/5_Word_Clouds.py [hunk 1]
```diff
@@ -152,7 +152,6 @@ def generate_single_wordcloud(word_freq: dict, title: str, colormap: str):
     plt.figure(figsize=(10, 5), dpi=100)
     plt.imshow(wordcloud, interpolation="bilinear")
     plt.axis("off")
-    plt.title(title, fontsize=20)
```
Comment (ayahusseini) [inline dashboard/pages/5_Word_Clouds.py:155]: Good quick change to make the dashboard more readable.

Comment (megansam5) [inline dashboard/pages/5_Word_Clouds.py:155]: Wow shes so good!

**Review by megansam5 — APPROVED** [review]
Impeccable coding skills!

**Review by ayahusseini — APPROVED** [review]

**Pull request merged**